django_pin_auth/forms.py

```diff
@@ -41,13 +41,13 @@ def prepare_form(self):
     def send_email(self, request):
```

Comment:
Line 38 raises a User matching query does not exist. when logging in with unused email.

Reply:
Is that intentional, for the library user to catch?

Reply:
No, it was meant to be caught at the form validation level.
Will fix